content/hls-2.10.0.0/index.md

```diff
@@ -0,0 +1,127 @@
++++
+title = "Haskell Language Server 2.10.0.0 release"
+date = 2025-04-07
+[taxonomies]
+authors = ["Fendor"]
+categories = ["HLS"]
+tags = ["Release"]
++++
+
+The HLS Team is proud to announce the new release of the Haskell Language Server `2.10.0.0`!
+
+<!-- more -->
+
+The last release of HLS was quite some time ago, back in July of 2024.
+Thus, this release has quite a number of new features, some of which we would like to highlight for you.
+
+## HLS `2.10.0.0`
+
+The release `2.10.0.0` supports the following GHC versions:
+
+* `9.12.2`
+* `9.10.1`
+* `9.8.4`
+* `9.6.7`
+* `9.4.8`
+
+We expect `2.10.0.0` to be the last HLS release having support for GHC `9.4` versions.
+
+### Go to Implementations
+
+We added support for a new LSP feature called `Go to Implementations`.
+This LSP feature allows you to jump to the implementation of overloaded functions.
+
+<img src="jump-to-implementations.gif" alt="A simple 'Demo' typeclass illustrating that the 'Go to Implementations request sends the cursor to the instance declaration 'Demo Char'." width=70%>
+
+### Inlay Hints
+
+Inlay hints allow you to display additional information within a line of code.
+This can be an elegant alternative to code lenses, which can be a bit more invasive in the editor.
```

Comment:
When you say that it's an alternative to code lenses, does it mean it's on by default?
If not, could you mention it here along with what setting to use to enable them?

Reply:
It depends on the plugin, import code lenses are replaced by inlay hints and if the editor doesn't support inlay hints (or they are disabled), HLS falls back to code lenses.

But it is a good point, perhaps this is something I should mention here as well.